content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -12,35 +12,73 @@ weight: 30
+  - Articles must contain content that applies broadly to the Kubernetes community - For example, a submission should focus on upstream Kubernetes as opposed to vendor-specific configurations.
+  - Links should primarily be to the official Kubernetes documentation. When using external references, links should be diverse - For example a submission shouldn't contain only links back to a single company's blog.
+  - This can sometime be a delicate balance. The [blog team](https://kubernetes.slack.com/messages/sig-docs-blog/) is there to give guidance on whether a post is appropriate for the Kubernetes blog, so don't hesistate to reach out. 
+- Blog posts do not offer deadlines.
+    - Articles are reviewed by community volunteers and _cannot guarantee_ any time-based deadlines, though we will try our best to accomodate you.
+  - Many core parts of the project are submitting posts during release windows and publication time will be delayed. Consider submitting during a quieter period of the release cycle.
+  - If you are looking for greater coordination on post release dates, coordinating with [CNCF marketing] is a more appropriate choice than submitting a blog post.
+  - Sometimes reviews can get backed up. If you feel your review isn't getting the attention it needs, you can reach out to the blog team via [this slack channel](https://kubernetes.slack.com/messages/sig-docs-blog/) to ask in real time. 
+- Blog posts should be relevant to Kubernetes users.
+  - Topics related to participation in or results of Kubernetes SIGs activities are always on topic (see the work in the [Upstream Marketing Team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/blog-guidelines.md#upstream-marketing-blog-guidelines) for support on these posts). 
+  - The components of Kubernetes are purposely modular, so tools that use existing integration points like CNI and CSI are on topic. 
+  - Posts about other CNCF projects may or may not be on topic. We recommend asking the blog team before submitting a draft.
+    - Many CNCF projects have their own blog. These are often a better choice for posts. There are times of major feature or milestone for a CNCF project that users would be interested in reading on the Kubernetes blog.
+- Blog posts should aim to be future proof
+  - Given the development velocity of the project, we want evergreen content, that won't require updates to stay accurate to the reader. 
+  - It can be a better choice to add a tutorial or update official documentation, then write a higher level overview as a blog post.
+    - Consider concentrating the long technical content as a call to action of the blog post, and focus on the problem space or why readers should care.
+
+### Technical Considerations for submitting a blog post
+
+Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are [many resources available] on how to use this technology stack.
```

Comment:
```suggestion
Submissions need to be in Markdown format to be used by the [Hugo](https://gohugo.io/) generator for the blog. There are many resources available on how to use this technology stack.
```

Reply:
Due to the third-party content suggestion, I hesitated to add in any links here, but we could link to things via the Hugo site or one of the "Awesome Project" repos, otherwise!

Reply:
Hugo is essential third party software in as much as the contents of the website repo aren't much use without Hugo.

Reply:
Good point - I think this qualifies as "...if it's necessary for Kubernetes to function."

Reply:
Updated with a link to the Hugo docs around getting started ✅